inc/api.php

```diff
@@ -229,6 +267,23 @@ public function add_watermark( $file ) {
 
 		return $response;
 	}
+	/**
+	 * Call the images endpoint.
+	 *
+	 * @param string|false $scroll_id Scroll id used to advance the search.
+	 * @return mixed The decoded json response from the api.
+	 */
+	public function get_cloud_images( $scroll_id = false ) {
+		$params = ['key' => Optml_Config::$key ];
+		if ( $scroll_id === false ) {
+			$scroll_id = get_transient( 'scroll_id' );
+			delete_transient( 'scroll_id' );
```

Comment:
I'm not sure if the transient is absolutely required here, however, if we need it we can use a prefix for it since transients are in the global scope. i.e `optimole_media_page_number`